test/functions.jl

```diff
@@ -3,43 +3,86 @@ using NamedDims: names
 using Test
 using Statistics
 
-@testset "sum" begin
-    nda = NamedDimsArray([10 20; 30 40], (:x, :y))
+@testset "Base" begin
+a = [10 20; 31 40]
```

Comment:
Code within the testset should be indented

Reply:
I agree in principle, the problem is that this file was already way over the line length, and indenting it further made writing the code on my little laptop screen almost impossible

I think a better solution is a follow-up PR that fixes up the style in the test - what do you think?

Reply:
That seems good to me

Reply:
Cool - i'll merge this and make a follow up PR for style changes :) thanks!